client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -24,20 +25,34 @@ const DesignSelector: FunctionComponent = () => {
 	const { selectedDesign, siteVertical } = useSelect( select =>
 		select( ONBOARD_STORE ).getState()
 	);
-	const { setSelectedDesign } = useDispatch( ONBOARD_STORE );
+	const { setSelectedDesign, resetOnboardStore } = useDispatch( ONBOARD_STORE );
+
+	const handleStartOverButtonClick = () => {
+		resetOnboardStore();
+	};
```

Comment:
This is the same as `const handleStartOverButtonClick = resetOnboardStore`, and I don't think the alias adds anything.

We can use it directly as the `onClick`:

```js
<Link onClick={ resetOnboardStore } >…</Link>
```

Reply:
Pretty sure that we'll need to add tracking here, so alias might actually prove useful?

Reply:
It's easy to add when we need it.